### PR TITLE
tweaked driver display settings to improve readability

### PR DESCRIPTION
**Pull request by crecine**
### Summary

Adjusted settings to improve readability

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/interface/methods_for_level2.py [hunk 1]
```diff
@@ -1604,8 +1604,10 @@ def add_driver(self, optimizer=None, use_coloring=None, max_iter=50, verbosity=V
         if verbosity != Verbosity.QUIET:
             if isinstance(verbosity, list):
                 driver.options['debug_print'] = verbosity
-            elif verbosity.value >= 2:
+            elif verbosity.value >= 3:
                 driver.options['debug_print'] = ['desvars', 'ln_cons', 'nl_cons', 'objs']
+        if verbosity is not Verbosity.DEBUG and optimizer in ("SNOPT", "IPOPT"):
```
Comment (Kenneth-T-Moore) [inline aviary/interface/methods_for_level2.py:1609]: Hang on, I think we want the pyoptsparse resulst for for verbose. and above. 

I would make it "if verbosity < Verbosity.VERBOSE"

File: aviary/interface/methods_for_level2.py [hunk 2]
```diff
@@ -1604,8 +1604,10 @@ def add_driver(self, optimizer=None, use_coloring=None, max_iter=50, verbosity=V
         if verbosity != Verbosity.QUIET:
             if isinstance(verbosity, list):
                 driver.options['debug_print'] = verbosity
-            elif verbosity.value >= 2:
+            elif verbosity.value >= 3:
```
Comment (Kenneth-T-Moore) [inline aviary/interface/methods_for_level2.py:1607]: For this one, "elif verbosity.value >= Verbosity.VERBOSE"
then turn on the driver debug.
Note, there are currently no options above VERBOSE, so you could just as well make it "== Verbosity.VERBOSE"

Comment (crecine) [reply aviary/interface/methods_for_level2.py:1607]: we have 4 options quiet (0), brief (1), verbose (2), and debug (3)

Comment (Kenneth-T-Moore) [reply aviary/interface/methods_for_level2.py:1607]: Sorry, I meant Verbosity.DEBUG everywhere I wrote VERBOSE above.

File: aviary/interface/methods_for_level2.py (hunk 1 above)
Comment (crecine) [reply aviary/interface/methods_for_level2.py:1609]: I responded on the teams, since I didn't see a way to add a collapsible comment here

**Review by Kenneth-T-Moore — APPROVED** [review]

**Review by ehariton — APPROVED** [review]

**Pull request merged**